packages/jest-changed-files/src/hg.ts

```diff
@@ -4,16 +4,14 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {Path} from 'types/Config';
-import type {Options, SCMAdapter} from 'types/ChangedFiles';
-
 import path from 'path';
 import execa from 'execa';
 
-const env = {...process.env, HGPLAIN: 1};
+import {Path, Options, SCMAdapter} from './types';
+
+const env = {...process.env, HGPLAIN: '1'};
```

Comment:
I did this:

```diff
- const env = {...process.env, HGPLAIN: 1};
+ const env = {...process.env, HGPLAIN: '1'};
```
because the type of `NodeJS.ProcessEnv` is:

```javascript
{
  [key: string]: string | undefined;
}
```

which conflicts with `HGPLAIN: 1` which is a `number`

Reply:
Makes sense. TS is correct, everything is coerced to a string (including `undefined`, fwiw)